### DISCUSSION TITLE
Any plans to add Stabiity.AI StableCode

### DISCUSSION THREAD

**Discussion by bozzy01** [danny-avila/LibreChat#774] [Feature Requests & Suggestions]
This was just announced today, Stability.AI has created a LLM specific to coding. 

https://stability.ai/blog/stablecode-llm-generative-ai-coding

I'd love to see this added to the list of available models for use. Actually looking a bit deeper this one may not be a chat based option it may only integrate with IDEs.

**Comment by danny-avila** [comment]
Would love to integrate as many tools/LLMs as possible, that is the vision! I think currently this can be used with HuggingFace, and supporting huggingface models is in the project list.

Would also love to see using this as a plugin

**Comment by Dual-0** [comment]
Stability AI says developers can now access Stable Diffusion 3 from its [Developer Platform](https://platform.stability.ai/docs/api-reference?_gl=1*1elp9gl*_ga*MTY1NDUzMTIwNC4xNzA3OTMwOTU4*_ga_W4CMY55YQZ*MTcxMzM2MjEzNy40LjEuMTcxMzM2MjU5Mi4wLjAuMA..#tag/Generate/paths/~1v2beta~1stable-image~1generate~1sd3/post).

source: https://www.theverge.com/2024/4/17/24132927/stable-diffusion-3-api-availability-stability-ai

would be great to have this in LibreChat
